src/wallet.py

```diff
@@ -0,0 +1,92 @@
+import json
+from json.decoder import JSONDecodeError
+
+import structlog
+from nacl.encoding import HexEncoder
+from nacl.signing import SigningKey
+import base64
+from cryptography.fernet import Fernet
+
+
+logger = structlog.get_logger(__name__)
+
+class wallet:
+    PRIVATE_KEY = ""
+    PUBLIC_KEY = ""
+    
+    def generate_wallet():
+        password = input("\nSe necesita una contraseña, escriba una: ")    
+        passws = password.encode('utf-8') #Se codifica para que lo admita
+        #Fernet pide una longitud de 32 bytes, a la contraseña que se usa
+        #se añaden tantos '0' hasta completarlo
+        while (len(passws) < 32):
```

Comment:
Perhaps this is better and in just 1 line?
```python
passws.ljust(32)
```

Reply:
I didn't know that, much better, thank you.